spec/requests/extra_locales_controller_spec.rb

```diff
@@ -31,54 +22,49 @@
       expect(response.status).to eq(400)
     end
 
-    context "when logged in as a moderator" do
-      let(:moderator) { Fabricate(:moderator) }
```

Comment:
Just removing the context here. Tests inside are unchanged. (best reviewed with whitespace-changes hidden)